internal/stages_test.go

```diff
@@ -103,11 +103,18 @@ func TestStages(t *testing.T) {
 			StdoutFixturePath:   "./test_helpers/fixtures/write_tree",
 			NormalizeOutputFunc: normalizeTesterOutput,
 		},
-		"pass_all": {
+		"pass_all_against_ryans_git": {
```

Comment:
We don't need two of these, let's just test against git and remove all that extra code